pkg/module_manager/module_manager.go

```diff
@@ -550,7 +554,42 @@ func (mm *moduleManager) Init() error {
 		log.Warnf("ConfigMap/%s has values for absent modules: %+v", app.ConfigMapName, unknownNames)
 	}
 
-	return nil
+	return mm.validateKubeConfig(mm.kubeConfigManager.CurrentConfig())
+}
+
+func (mm *moduleManager) validateKubeConfig(kubeConfig *kube_config_manager.Config) error {
```

Comment:
Method name looks confusing. At first, I thought, that it was related to kubeconfig somehow. Maybe it is just my ignorance :)

Reply:
It's a Config from Kubernetes =)